R/simulate.R

```diff
@@ -61,6 +61,7 @@ blank_umFit <- function(fit_function){
 
 setMethod("simulate", "character",
   function(object, nsim=1, seed=NULL, formulas, coefs=NULL, design, guide=NULL, ...){
+  set.seed(seed)
```

Comment:
My bad, I did not know it was required by the generic `simulate` function and thought it was an oversight because I didn't read the function documentation. I'll remove it.

Reply:
I removed it: https://github.com/rbchan/unmarked/pull/267/commits/6484766345ebd17c60c214917f0c86c4af569765#diff-b3a990c33cbe8493a526a29af4cb7c023d3da3e27a26389834739153c45d6065L64